core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -390,3 +390,38 @@ message SQLExecutionUIData {
   repeated int64 stages = 11;
   map<int64, string> metric_values = 12;
 }
+
+message RDDOperationEdge {
+  int32 from_id = 1;
+  int32 to_id = 2;
+}
+
+enum DeterministicLevel {
+  DETERMINATE = 0;
```

Comment:
Let's add `UNSPECIFIED` as field 0.
See discussion in https://github.com/apache/spark/pull/38779#discussion_r1032062307

Reply:
makes sense, but getting this error,
> store_types.proto:400:3: "UNSPECIFIED" is already defined in "org.apache.spark.status.protobuf".
> 
> 
> store_types.proto:400:3: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "UNSPECIFIED" must be unique within "org.apache.spark.status.protobuf", not just within "DeterministicLevel".

Reply:
try moving the enum into RDDOperationGraphWrapper ?